### DISCUSSION TITLE
Vite hot update is invalid

### DISCUSSION THREAD

**Discussion by ZhongboHuang** [tailwindlabs/tailwindcss#6657] [Help]
I use the latest version of tailwindcss in a vite2 project.But when i change some class name, the vite's HMR is invalid.I must restart the project.

**Comment by fourwhitesocks** [comment]
I updated a SvelteKit project from tw2 to tw3 and experiencing the same issue...I followed the TW upgrade docs completely, I was thinking it was an issue with my version of Vite maybe??  When I restart the dev server then the edited classes appear.  I opened a similar thread a few days before you did but there are no responses yet.

**Reply by baihuijun** [reply]
我也是遇到这个问题,每次改完都得重启,请问解决了吗
